src/kibana/styles/main.less

```diff
@@ -203,7 +203,16 @@ notifications {
     z-index: 2 !important;
   }
 }
-
+.lr-custom-search-less-than-equal:before {
+  content: "\2264";
```

Comment:
(Yannick explained that they were unicode characters). 

@yannick-polius please put a comment in that explains this